docs(respect): add notes about severity configuration (#2025)
Redocly/redocly-cli: docs/commands/respect.md

  You can also pass the server overrides as an environment variable, as in the following example:
  `REDOCLY_CLI_RESPECT_SERVER="test=https://test.com"`.

---

- --severity
- string
- Severity of the check.

>   Customize the severity level of validation checks for status codes, schemas, and content types. This is useful when:

Reviewer: ```suggestion
  Customize the severity level of validation checks for status codes, schemas, and content types. Setting the severity level is useful when:
```

>   - You're actively developing your API and want to temporarily relax validation

Reviewer: ```suggestion
  - You're actively developing your API and want to temporarily relax validation.
```

>   - You're testing new API behavior that doesn't match the current schema

Reviewer: ```suggestion
  - You're testing new API behavior that doesn't match the current schema.
```

>   - You've encountered known issues that shouldn't block test execution

Reviewer: ```suggestion
  - You've encountered known issues that shouldn't block test execution.
```

>   - You want to treat certain validation failures as warnings instead of errors

Reviewer: ```suggestion
  - You want to treat certain validation failures as warnings instead of errors.
```

>   The following checks can be configured: `STATUS_CODE_CHECK`, `SCHEMA_CHECK`, and `CONTENT_TYPE_CHECK`.

Reviewer: Isn't there a successCriteriaCheck? Unless this is only for Realm? Also, is the format different from Realm - statusCodeCheck?

Author: Yes, please add `SUCCESS_CRITERIA_CHECK`

Author: Format of cli commands is different, see the example.
Reunite has its own  config to value mapping  when you configure severity for API Monitoring.

  The following severity values are available:

  - `error`: Validation failures cause the workflow to fail

>   - `warn`: Validation failures appear as warnings but don't fail the workflow

Reviewer: ```suggestion
  - `warn`: Validation failures appear as warnings but don't fail the workflow.
```

  - `off`: Validation checks still run but their results are ignored

>   Note that setting severity to `off` does not prevent checks from executing - it only prevents their results from affecting the workflow outcome.

Reviewer: ```suggestion
{% admonition type="info" %}
Setting the severity of a check to `off` does not prevent the check from executing - it only prevents the check's results from affecting the workflow outcome.
{% /admonition %}
```

>   For example, the following command turns status code errors into warnings:

Reviewer: ```suggestion
  For example, the following command sets status code errors to the warning severity level:
```
